test/integration/amp-handler-test.js

```diff
@@ -6,184 +8,193 @@ const assert = require("assert");
 const express = require("express");
 
 const ampConfig = {
-  fonts: {
-    primary: {
-      url: "Open+Sans:300,400,600,700",
-      family: '"Open Sans", sans-serif',
+  ampConfig: {
+    fonts: {
+      primary: {
+        url: "Open+Sans:300,400,600,700",
+        family: '"Open Sans", sans-serif',
+      },
+      secondary: {
+        url: "PT+Serif:400,400italic,700,700italic",
+        family: "sans-serif",
+      },
     },
-    secondary: {
-      url: "PT+Serif:400,400italic,700,700italic",
-      family: "sans-serif",
+    colors: {
+      primary: "#294f32",
+      secondary: "#763194",
+      "footer-background": "#FDBD10",
+      "section-text-color": "#f4e842",
     },
+    "invalid-elements-strategy": "redirect-to-web-version",
+    "related-collection-id": 1234,
   },
-  colors: {
-    primary: "#294f32",
-    secondary: "#763194",
-    "footer-background": "#FDBD10",
-    "section-text-color": "#f4e842",
-  },
-  "invalid-elements-strategy": "redirect-to-web-version",
-  "related-collection-id": 1234,
 };
 
```

Comment:
change to `getClientStub` was needed because I wanted to configure what `client.getConfig` returned. `getClientStub` now accepts `getHostname`, `getConfig` etc. as params. So I can pass a different `getConfig` to it